### PR TITLE
Accessible modals

### PR DESCRIPTION
**Pull request by voluntadpear**
This PR improves the accessibility of our modals (the color picker and the export dialog)

It implements a focus trap so that tapping through the controls inside them don't escape to outer elements, it also allows to close the modals with the "Escape" key.

### CODE REVIEW

File: src/actions/actionExport.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ export const actionChangeProjectName: Action = {
   },
   PanelComponent: ({ appState, updateData }) => (
     <EditableText
+      label="File title"
```
Comment (lipis) [inline src/actions/actionExport.tsx]: Shall we use a new section in the translations?

Comment (voluntadpear) [reply src/actions/actionExport.tsx]: Yes! I'm thinking of sending those in a follow-up. But I can do it in the current PR if that would be better.

**Comment by lipis** [comment]
Yes I think we can do it in go

**Comment by voluntadpear** [comment]
I added the translations strings and also applied the Spanish ones (since it's my mother language  🙂 )

**Pull request merged**